tob_metadata_harmonisation.py

```diff
@@ -0,0 +1,252 @@
+"""
+This script is used to harmonise the assay metadata from the old tob-wgs project with the new metadata schema.
+This script checks the assay metadata for each assay in the project and checks if the field is in the FIELD_MAPPING dictionary.
+If the field is in the dictionary, the field is added to the new_assay_data dictionary with the new field name as the key.
+If the field is not in the dictionary, the a ValueError is raised.
+If the field is in the dictionary but the value is None, the field is not added to the new_assay_data dictionary.
+If the field is in the dictionary and the value is not None, the field is added to the new_assay_data dictionary with the new field name as the key and the value as the value.
+If there are multiple mappings to the same field, the value is compared to the value already in the new_assay_data dictionary. If the values are different, the field is printed to the console.
+"""
+from typing import Dict
+
+from metamist.graphql import gql, query
+
+from metamist.apis import AssayApi
+from metamist.models import AssayUpsert
+
+
+ACTIVE_SG_QUERY = """
+query MyQuery {
+  sample(project: {eq: "tob-wgs-test"}) {
+    id
+    sequencingGroups(activeOnly: {eq: true}) {
+      id
+      assays {
+        id
+        meta
+      }
+    }
+  }
+}
+"""
+
+
+INACTIVE_SG_QUERY = """
+query MyQuery {
+  sample(project: {eq: "tob-wgs-test"}) {
+    id
+    sequencingGroups(activeOnly: {eq: false}) {
+      id
+      assays {
+        id
+        meta
+      }
+    }
+  }
+}
+"""
+SG_ASSAY_QUERY = """
```

Comment:
this is also a duplicate query, but with neither true or false specified? I'm still working through this, but could this either be replaced with the two individual true/false calls, or could you just use one query for everything where `active` is returned as an attribute, then filter the results for true/false as required

Reply:
Just checked, this isn't used at all - should probably be removed

Reply:
I found out that if you have `activeOnly:{}` without a boolean in there, it returns both `active` and `inactive`

But yes, you're right, this can be removed